itest/loadtest/mint_batch_test.go

```diff
@@ -194,7 +197,22 @@ func mintBatchStressTest(t *testing.T, ctx context.Context,
 			},
 		},
 	)
-	require.NoError(t, err)
+	if err != nil {
+		// Only fail the test for duplicate universe errors, as we might
+		// have already added the server in a previous run.
+		require.ErrorContains(
+			t, err, universe.ErrDuplicateUniverse.Error(),
+		)
```

Comment:
Do the the comment and `require` statement align here?

> fail the test for duplicate universe errors

But then the test pass if the error contains `universe.ErrDuplicateUniverse`. I think the comment should be:
```
		// Only fail the test if the error is not a duplicate universe error, as we might
		// have already added the server in a previous run.
```

Reply:
Ah, you're right, wanted to write `Only fail the test for other errors than duplicate universe errors`, fixed.